lib/remote_console/client_adapter/web_mks.rb

```diff
@@ -5,6 +5,7 @@ class WebMKS < SSLSocket
 
```

Comment:
I generally like to have a description for each `initialize` method. Describe what the arguments are for, what it returns, and what its general purpose is. Is this something only meant to be used internally, for example? That sort of thing.

Reply:
Well, all the initialize methods are the same across the adapters, so I think a single definition might be more helpful than a redundant one.